modules/mw.cx.TranslationMdwiki.js

```
@@ -199,124 +227,202 @@ async function get_from_medwiki_or_mdwiki_api(title, tr_type) {
 async function get_mdtexts_2024(title) {
-	let sanitizedTitle = title.replace(/['" :/]/g, "_");
-
-	// const url = "https://medwiki.toolforge.org/mdtexts/segments.php?title=" + sanitizedTitle;
-	const url = `/mdtexts/segments.php?title=${sanitizedTitle}`;
-
-	const data = await fetchGetJson(url);
-	if (!data) {
-		console.log("get_mdtexts_2024: not found");
-		return false;
-	}
+	var title = title.replace(/['" :/]/g, "_");
```

Biome (1.9.4)

[error] Shouldn't redeclare 'title'. Consider to delete it or rename it.

'title' is defined here:


(lint/suspicious/noRedeclare)